.github/workflows/ci.yml

```
@@ -0,0 +1,44 @@
+name: CI
+ 
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    types: [opened, synchronize]
+ 
+jobs:
+  build:
+      name: Build and Test
+      timeout-minutes: 15
+      runs-on: ubuntu-latest
+      # To use Remote Caching, uncomment the next lines and follow the steps below.
+      # env:
+      #  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+      #  TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+ 
+      steps:
+        - name: Check out code
+          uses: actions/checkout@v3
+          with:
+            fetch-depth: 2
+ 
+        - name: Setup Node.js environment
+          uses: actions/setup-node@v3
+          with:
+            node-version: 18
+            cache: 'yarn'
+ 
+        - name: Install dependencies
+          run: yarn install
+
+        - name: Generate Prisma Types
+          run: yarn run db:generate
+ 
+        - name: Build
+          run: yarn build
+          env:
+            SKIP_ENV_VALIDATION: true
+            EMAIL_FROM: envless@example.com
+ 
+        - name: Test
```

```suggestion
        - name: Unit Tests
          run: yarn test

        - name: E2E Tests
          run: yarn test:e2e
```

Lets make these changes and delete both of these actions

* https://github.com/envless/envless/blob/main/.github/workflows/playwright.yml
* https://github.com/envless/envless/blob/main/.github/workflows/vitest.yml

---

what about linting one ? @dahal I missed this job does it have to be written ?